hackclub.com.yaml

```diff
@@ -29,6 +29,20 @@
       - v=spf1 include:_spf.google.com include:mailgun.org include:amazonses.com include:sendgrid.net ~all
       - stripe-verification=5bdd0e2475b12744bfe1f778ac419f917800d476fd498ae2682ecd161d162c64
       - 1password-site-verification=Q4UYPWIJSBDT5KWEECIQPRAZJ4 # For 1password's breach notification feature
+
+
+ihs:
+  - ttl: 1
+    type: CNAME
+    value: https://ihshack.vercel.app/.
+
+
+_vercel:
```

Comment:
FYI: we already have a `_vercel` record. Could you add your TXT record to that? Thank you so much!

Reply:
I added the TXT record to the already existing _vercel record.